crates/nu-cli/src/completions/file_completions.rs

```diff
@@ -157,3 +151,30 @@ pub fn matches(partial: &str, from: &str) -> bool {
     from.to_ascii_lowercase()
         .starts_with(&partial.to_ascii_lowercase())
 }
+
+// escape paths that contains some special characters
+pub fn escape_path_str(path: String) -> String {
+    let mut path = path;
+
+    // Check if path needs to be escaped
+    let needs_escape = path.bytes().fold(
+        false,
+        |acc, x| {
+            acc
+        || (x >> 4) == 0b0010
+        || x == b'\\' // 0x5c
+        || x == b'`'
+        }, // 0x60
```

Comment:
Why is this comment not on the line it describes?

Reply:
Good question, probably the formatter. Fixed